guides/common/modules/con_using-external-databases.adoc

```diff
@@ -2,4 +2,4 @@
 = Using external databases with {Project}
```

Comment:
Is this module still used somewhere? From what I can see, I think you can delete it and also `assembly_using-external-databases.adoc`.

Reply:
Moving the "PostgreSQL as an external database considerations" module might be needed, though. I'll take a stab at that in the next iteration.